docs/contribute/experimentalist-module.md

```diff
@@ -0,0 +1,128 @@
+  a subset of conditions from the pool to be used in the next experiment.
+- **Samplers** directly return a subset of experimental conditions from a pool of candidate experimental conditions.
+
+## Repository Setup
+
+We recommend using the [cookiecutter template](https://github.com/AutoResearch/autora-template-cookiecutter) to set up
+a repository for your experimentalist. Alternatively, you use the 
+[unguided template](https://github.com/AutoResearch/autora-template). If you are using the cookiecutter template, you can set up your repository using
+
+```shell
+cookiecutter https://github.com/AutoResearch/autora-template-cookiecutter
+```
+
+Make sure to select the `experimentalist` option when prompted. You may also select whether you want to implement an experimentalist as a sampler, pooler, or custom function. You can skip all other prompts pertaining to other modules 
+(e.g., experiment runners) by pressing enter.
+
+## Implementation
+
+Irrespective of whether you are implementing a pooler or a sampler, 
+you would want to implement a function that returns a set of experimental conditions. This set may be
+a numpy array, iterator variable or other data format. 
+
+!!! hint
+    We generally **recommend using 2-dimensional numpy arrays as outputs** in which
+    each row represents a set of experimental conditions. The columns of the array correspond to the independent variables.
+
+### Implementing poolers
+
+Once you've created your repository, you can implement your experimentalist pooler by editing the `init.py` file in 
+``src/autora/experimentalist/pooler/name-of-your-experimentalist/``. 
+You may also add additional files to this directory if needed. 
+It is important that the `init.py` file contains a function called `name-of-your-experimentalist` 
+which returns a pool of experimental conditions (e.g., as an iterator object or numpy array).
+
+The following example ``init.py`` illustrates the implementation of a simple experimentalist pooler
+that generates a grid of sampels within the specified bounds of each independent variable (IV):
+
+```python 
+
+"""
+Example Experimentalist Pooler
+"""
+
+from itertools import product
+from typing import List
+from autora.variable import IV
+
+
+def grid_pool(ivs: List[IV]):
+    """
+    Creates exhaustive pool from discrete values using a Cartesian product of sets
+
+    Arguments:
+        ivs {List[IV]}:  List of independent variables
+
+    Returns:
+        pool: An iterator over all possible combinations of IV values
+    """
+
+    l_iv_values = []
+    for iv in ivs:
+        assert iv.allowed_values is not None, (
+            f"gridsearch_pool only supports independent variables with discrete allowed values, "
+            f"but allowed_values is None on {iv=} "
+        )
+        l_iv_values.append(iv.allowed_values)
+
+    # Return Cartesian product of all IV values
+    return product(*l_iv_values)
+
+
+```
+
+### Implementing samplers
```

Comment:
this is super nitpicky, but is there logic to when we use title case vs sentence case in headings?

Reply:
Good point. I think we should generally stick with sentence case. What do you think?

Reply:
sounds good to me. addressed in this file for this PR, and will address globally in a separate PR